packages/office-ui-fabric-react/src/components/Image/Image.tsx

```diff
@@ -124,21 +112,24 @@ export class Image extends BaseComponent<IImageProps, IImageState> {
     );
   }
 
-  private _evaluateImage(): boolean {
-    let { src } = this.props;
+  @autobind
+  private _onImageLoaded(ev: React.SyntheticEvent<HTMLImageElement>): boolean {
+    let { src, onLoad } = this.props;
     let { loadState } = this.state;
-    let isLoaded = (src && this._imageElement.naturalWidth > 0 && this._imageElement.naturalHeight > 0);
+
+    if (onLoad) {
+      onLoad(ev);
+    }
 
     this._computeCoverStyle(this.props);
 
-    if (isLoaded && loadState !== ImageLoadState.loaded && loadState !== ImageLoadState.errorLoaded) {
-      this._events.off();
-      this.setState({
+    if (src && loadState !== ImageLoadState.loaded && loadState !== ImageLoadState.errorLoaded) {
+        this.setState({
```

Comment:
im very confused by this function. I also think by removing recomputing on updates, you may be introducing a bug when the image is cached and loaded in IE.